### PR TITLE
Add example of a picture based pipeline and 2D DisplayHandle

### PR DESCRIPTION
**Pull request by alesgenova**

### CODE REVIEW

File: examples/operators/gaussian_blur/run.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from oremda.typing import JSONType, PortKey, RawPort, DisplayType
+
+
+@operator
+def gaussian_blur(
+    inputs: Dict[PortKey, RawPort], parameters: JSONType
+) -> Dict[PortKey, RawPort]:
+
+    sigma = parameters.get('sigma', 0)
+    data = inputs['in'].data
```
Comment (cclauss) [inline examples/operators/gaussian_blur/run.py:16]:
```suggestion
    data = inputs['in'].data or gaussian_filter(data, sigma=sigma)
```

File: server/server/main.py [hunk 2]
```diff
@@ -175,7 +175,9 @@ def notify(message: NotificationMessage):
 
     def display_factory(id: IdType, display_type: DisplayType):
         if display_type == DisplayType.OneD:
-            return OneDDisplayHandle(id, notify)
+            return DisplayHandle1D(id, notify)
+        if display_type == DisplayType.TwoD:
```
Comment (cclauss) [inline server/server/main.py:179]:
```suggestion
        elif display_type == DisplayType.TwoD:
```

Comment (alesgenova) [reply server/server/main.py]: Thanks for catching this, I fixed it!

**Review by psavery — APPROVED** [review]
LGTM. Can you fix the CI?

I think it would also be nice if we eventually moved the matplotlib display handles into their own files somewhere to be re-used. I'm also using them in one of the MPI runner code examples.

@cjh1 Do you have any comments?

File: server/server/displays/__init__.py [hunk 3]
```diff
@@ -65,3 +65,54 @@ def render(self):
 
         message = generic_message(ActionType.DisplayRender, payload)
         self.notify(message)
+
+
+class DisplayHandle2D(DisplayHandle):
```
Comment (cjh1) [inline server/server/displays/__init__.py:70]: Am I right in the thinking this is a remote display handle, as it proxies the requests to the client? We might want to update the name to indicate this?

**Review by cjh1 — COMMENTED** [review]
Lets get the CI work then this can go in.

**Pull request merged**